library/src/main/java/com/novoda/downloadmanager/lib/DownloadsUriProvider.java

```diff
@@ -59,14 +55,6 @@ private static DownloadsUriProvider newInstance() {
     }
 
     /**
-     * The content URI for accessing publicly accessible downloads (i.e., it requires no
-     * permissions to access this downloaded file)
-     */
-    public Uri getPubliclyAccessibleDownloadsUri() {
```

Comment:
Oh wait, I might misunderstood the scope of this class: is this to be used by the client?